src/styles/components/features.scss

```diff
@@ -0,0 +1,63 @@
+.features {
+  margin-bottom: 128px;
+
+  &-about {
+    text-align: center;
+    margin-top: 90px;
+
+    &__text {
+      font-family: $fontNunito;
+      font-weight: 600;
+      font-size: $fontSizeM;
+      color: $primary;
+      margin: 0;
+    }
+
+    &__title {
+      font-family: $fontMont;
+      font-weight: 900;
+      font-size: $fontSizeXL;
+      color: $primary;
+      margin: 10px 0 0;
+    }
+  }
+
+  &-cards {
+    display: flex;
+    width: 1104px;
+    margin: 72px auto 0;
+
+    &-row {
+      display: block;
+      text-align: center;
+      width: 226px;
+      margin-right: 68px;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      &__img {
+        width: 80px;
+        height: 81px;
```

Comment:
Strange height, why is it bigger on 1px than width?

Reply:
fixed)